### PR TITLE
#185172112 implement reviews and feedback option

### PR DESCRIPTION
**Pull request by JosephNgabo**
#### What does this PR do?
Lets users to post the feedbacks and reviews on certain product
#### Description of Task to be completed?
after reveiwing single product their form that holds recent feedbacks added on product and can add anotheer new one.
#### How should this be manually tested?
by accessing the single product page
#### What are the relevant pivotal tracker/Trello stories?
#185172112

### CODE REVIEW

File: src/pages/feebdacks/tests/review.test.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    );
+    await waitFor(() => {
+      expect(screen.getByText('Top Reviews')).toBeInTheDocument();
+    });
+  });
+
+  test("Test the rating star to be in the document!", async () => {
+    const id = '6ac24373-0bc0-414b-92d2-d78f8ada8878'
+    render(
+      // <MemoryRouter initialEntries={['/product/6ac24373-0bc0-414b-92d2-d78f8ada8878']}>
```
Comment (AdrineUWERA) [inline src/pages/feebdacks/tests/review.test.tsx:35]: Remove the comments in this file.

**Review by AdrineUWERA — APPROVED** [review]

**Review by princoo — APPROVED** [review]

File: src/pages/feebdacks/tests/review.test.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
```
Comment (Pacific911) [inline src/pages/feebdacks/tests/review.test.tsx:35]: you can remove these comments if not needed

File: src/pages/feebdacks/tests/review.test.tsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      <Provider store={store}>
+        <Reviews productId={id}/>
+      </Provider>
+      // </MemoryRouter>
```
Comment (Pacific911) [inline src/pages/feebdacks/tests/review.test.tsx:39]: and here too

File: src/pages/feebdacks/tests/review.test.tsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    );
+    await waitFor(() => {
+      const btns = screen.getAllByText("Add review");
+      expect(btns.length).toBeGreaterThan(0);
+      btns.forEach(btn=>{
+        fireEvent.click(btn)
+        // expect(AddReview).toHaveBeenCalledWith(id);
```
Comment (Pacific911) [inline src/pages/feebdacks/tests/review.test.tsx:46]: check here

File: src/pages/feebdacks/tests/review.test.tsx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+      <Provider store={store}>
+        <Reviews />
+      </Provider>
+    );
+
+    const addReviewButton = getByText('Add review');
+
+    expect(addReviewButton).toBeInTheDocument();
+  });
+  // it('should dispatch reviews action correctly', async () => {
```
Comment (Pacific911) [inline src/pages/feebdacks/tests/review.test.tsx:73]: and on this entire file too if your not using them anywhere

**Review by Pacific911 — CHANGES_REQUESTED** [review]

**Review by Pacific911 — APPROVED** [review]

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**